ports/psoc6/mpconfigport.h

```diff
@@ -69,9 +59,11 @@
 #define MICROPY_READER_VFS                      (1)
 #define MICROPY_ENABLE_GC                       (1)
 #define MICROPY_ENABLE_EMERGENCY_EXCEPTION_BUF  (1)
+
 #define MICROPY_LONGINT_IMPL                    (MICROPY_LONGINT_IMPL_MPZ)
-// #define MICROPY_FLOAT_IMPL                      (MICROPY_FLOAT_IMPL_FLOAT)
-#define MICROPY_FLOAT_IMPL                      (MICROPY_FLOAT_IMPL_DOUBLE)
+#define MICROPY_FLOAT_IMPL                      (MICROPY_FLOAT_IMPL_FLOAT)
+// #define MICROPY_FLOAT_IMPL                      (MICROPY_FLOAT_IMPL_DOUBLE)
```

Comment:
??

Reply:
Python uses double precision, which is preferable. But there are bugs in the MPY code which should be fixed before using it. Also, at least 1 test fails because of rounding when using double prec, and 1 fails when using single prec (the comparison to Python, namely).
So what we have to discuss is how to move forward with helping to make MPY a better tool. I have found several issues, but for me it depends on how we continue with MPY.